### ISSUE TITLE
Turn off circular LEDs by default

### ISSUE THREAD

**Issue by bastiaanterhorst** [respeaker/usb_4_mic_array#52]
Hi, I'm implementing the 4 mic USB mic array v2, and I have a question about the pixel ring. I'd like to turn it off, especially when my device is still booting up. However, by default the pixel ring is lit, and also shows DOA. This is super bright and interfering with my design.

I know I can turn the ring off using the pixel-ring library, but that is only possible after boot.

Is there any way to adjust the firmware or change settings on the mic itself so that the ring is OFF by default?

I am using the latest v3 firmware (1 channel version).

Thanks!

**Comment by benedikt-bartscher** [comment]
we really need this!

**Comment by bastiaanterhorst** [comment]
@benedikt-bartscher I managed to solve the issue for me by flashing using the firmware linked on the forum here: https://forum.seeedstudio.com/t/respeaker-micarray-v2-0-how-to-turn-led-off/7079/2

I have no idea what version it is, and it is a pity a version with lights off isn't offered here on the main GH page, but for me this version works well.

**Comment by benedikt-bartscher** [comment]
@bastiaanterhorst Thank you very much, i wasn't aware of this firmware.

However, i am missing this feature in the current Firmware. A setting which persists over reboots would be awesome!

**Comment by bastiaanterhorst** [comment]
@benedikt-bartscher Yeah like I said, I don't know which version of the firmware is linked on the forum. But I can confirm it works well, including persisting over reboots.
